pontoon/base/migrations/0004_make_old_projects_public_again.py

```diff
@@ -0,0 +1,19 @@
+# Generated by Django 2.2.11 on 2020-05-16 22:22
+
+from django.db import migrations
+
+
+def make_projects_public(apps, schema_editor):
+    """Old projects were public by default."""
+
+    Project = apps.get_model("base", "Project")
+    Project.objects.update(visibility="public")
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("base", "0003_auto_20200505_1808"),
```

Comment:
I'm a fan of verbose migration names, so I'd suggest to rename migration `0003` to something more meaningful than `auto_XYZ`.